rdmd.d

```diff
@@ -916,3 +891,52 @@ void yap(size_t line = __LINE__, T...)(auto ref T stuff)
     debug stderr.writeln(line, ": ", stuff);
     else stderr.writeln(stuff);
 }
+
+/**
+Used to wrap filesystem operations that should also be logged with the
+yap function. These functions allow the the filenane to be given once
+so the log statement always matches the operation. Using it also guarantees
+you won't forget to include a `yap` alongside any file operation you want
+to be logged.
+*/
+static struct Yap
```

Comment:
`static` on non-templated top-level structs doesn't mean anything.

Reply:
I still think this is a poor name.

Reply:
Lol, I thought your `static:` please comment was talking about this struct...I was a bit confused, but I don't want to argue things that I don't really care about.